docs/root/start/_include/envoy-dynamic-demo.yaml

```diff
@@ -0,0 +1,49 @@
+node:
```

Comment:
High level comment: in a future change would it be good to add YAML comments to the config to help users? This applies elsewhere also.

Reply:
ok, noted